src/scripts/chart/column.js

```diff
@@ -49,7 +61,7 @@ function columnChart(g){
 	'use strict';
 
 	var model = new DataModel(Object.create(g.data()[0]));
-	var i = model.y.series.length;
+	var i = 0;
```

Comment:
re: comment from Ben - he was following crockford's rule about putting all vars above and js doesn't have block scope! http://javascript.crockford.com/code.html Got me on that one!